src/main/java/app/bpartners/geojobs/model/parcelization/area/AreaComputer.java

```diff
@@ -6,7 +6,7 @@
 public class AreaComputer implements Function<Geometry, Area> {
 
   @Override
-  public Area apply(Geometry geometry) {
+  public SquareDegree apply(Geometry geometry) {
     return new SquareDegree(geometry.getArea());
```

Comment:
So here, the Geometry contains already an `area` attributes ? And it is compute auto by the Geometry class ?